src/js/components/help/downloads.js

```diff
@@ -0,0 +1,324 @@
+      title
+    }
+  ];
+};
+
+const defaultCurlDownload = ({ location, tokens }) => {
+  let headers = `-H cookie: JWT=${getToken()}`;
+  if (tokens.length) {
+    headers = `-H Authorization: Bearer ${tokens[0]}`;
+  }
+  return `curl ${headers} -LO ${location}`;
+};
+
+const defaultGitlabJob = ({ location, tokens }) => {
+  const filename = location.substring(location.lastIndexOf('/') + 1);
+  return `
+download:mender-tools:
+  image: curlimages/curl
+  stage: download
+  variables:
+    ${tokens.length ? `MENDER_TOKEN: ${tokens}` : `MENDER_JWT: ${getToken()}`}
+  script:
+    - if [ -n "$MENDER_TOKEN" ]; then
+    - curl -H Authorization: Bearer $MENDER_TOKEN -LO ${location}
+    - else
+    - ${defaultCurlDownload(location)}
+    - fi
+  artifacts:
+    expire_in: 1w
+    paths:
+      - ${filename}
+`;
+};
+
+const tools = [
+  {
+    id: 'mender',
+    packageId: 'mender-client',
+    title: 'Mender Client package',
+    getLocations: multiArchLocationFormatter,
+    canAccess: () => true,
+    osList: defaultOSVersions,
+    archList: defaultArchitectures
+  },
+  {
+    id: 'mender-artifact',
+    title: 'Mender Artifact',
+    getLocations: defaultLocationFormatter,
+    canAccess: () => true,
+    osList: ['darwin', 'linux']
+  },
+  {
+    id: 'mender-binary-delta',
+    title: 'Mender binary delta',
+    getLocations: nonOsLocationFormatter,
+    isPublic: false,
+    canAccess: ({ isEnterprise, tenantCapabilities }) => isEnterprise || tenantCapabilities.canDelta
+  },
+  {
+    id: 'mender-cli',
+    title: 'Mender CLI',
+    getLocations: defaultLocationFormatter,
+    canAccess: () => true,
+    osList: ['darwin', 'linux']
+  },
+  {
+    id: 'mender-configure-module',
+    packageId: 'mender-configure',
+    title: 'Mender configure',
+    getLocations: multiArchLocationFormatter,
+    canAccess: ({ tenantCapabilities }) => tenantCapabilities.hasDeviceConfig,
+    osList: defaultOSVersions,
+    archList: ['all']
+  },
+  {
+    id: 'mender-connect',
+    title: 'Mender connect',
+    getLocations: multiArchLocationFormatter,
+    canAccess: () => true,
+    osList: defaultOSVersions,
+    archList: defaultArchitectures
+  },
+  {
+    id: 'mender-convert',
+    title: 'Mender convert',
+    getLocations: ({ versionInfo }) => [
+      {
+        location: `https://github.com/mendersoftware/mender-convert/archive/refs/tags/${getVersion(versionInfo, 'mender-convert')}.zip`,
+        title: 'Mender convert'
+      }
+    ],
+    canAccess: ({ tenantCapabilities }) => tenantCapabilities.hasDeviceConnect
+  },
+  {
+    id: 'mender-gateway',
+    title: 'Mender gateway',
+    getLocations: multiArchLocationFormatter,
+    isPublic: false,
+    canAccess: ({ isEnterprise }) => isEnterprise,
+    osList: defaultOSVersions,
+    archList: defaultArchitectures
+  },
+  {
+    id: 'monitor-client',
+    packageId: 'mender-monitor',
+    title: 'Monitor-client',
```

Comment:
To be consistent with the other names above...

```suggestion
    title: 'Mender Monitor',
```